.github/workflows/cmake.yml

```diff
@@ -113,29 +113,29 @@ jobs:
           # BerkeleyDB: Using our own FindBerkeleyDB module.
           - system_packages: 'libboost-dev libevent-dev'
             depends_options: 'NO_BDB=1 NO_SQLITE=1 NO_USDT=1'
-            build_options: '-DWITH_BDB=ON'
+            build_options: '-DENABLE_WALLET=ON -DWITH_BDB=ON'
             expected: 'fail'
           - system_packages: 'libboost-dev libevent-dev libdb-dev libdb++-dev'
             depends_options: 'NO_BDB=1 NO_SQLITE=1 NO_USDT=1'
-            build_options: '-DWITH_BDB=ON'
+            build_options: '-DENABLE_WALLET=ON -DWITH_BDB=ON'
             expected: 'fail'
           - system_packages: 'libboost-dev libevent-dev'
             depends_options: 'NO_SQLITE=1 NO_USDT=1'
-            build_options: '-DWITH_BDB=ON'
+            build_options: '-DENABLE_WALLET=ON -DWITH_BDB=ON'
             expected: 'pass'
 
           # SQLite: Using CMake's FindSQLite3 module.
           - system_packages: 'libboost-dev libevent-dev'
             depends_options: 'NO_BDB=1 NO_SQLITE=1 NO_USDT=1'
-            build_options: '-DWITH_SQLITE=ON'
+            build_options: '-DENABLE_WALLET=ON -DWITH_SQLITE=ON'
```

Comment:
Shouldn't `WITH_SQLITE=ON` imply `ENABLE_WALLET`? What (extra) does `ENABLE_WALLET` do here? I'm not sure we'll actually want to keep it as an option.

Reply:
Reworked in https://github.com/hebasto/bitcoin/pull/164.